src/lib/storage/dictionary_column.cpp

```diff
@@ -21,6 +21,11 @@ DictionaryColumn<T>::DictionaryColumn(const pmr_vector<T>&& dictionary,
     : _dictionary(std::make_shared<pmr_vector<T>>(std::move(dictionary))), _attribute_vector(attribute_vector) {}
 
 template <typename T>
+DictionaryColumn<T>::DictionaryColumn(const std::shared_ptr<pmr_vector<T>>& dictionary,
```

Comment:
We need this for `DictionaryColumn::migrate` where a new shared_ptr is created. If we only pass the dereferenced shared_ptr it would be deallocated when it goes out of scope (ie. immediately).